dags/resources/stages/sessions/sqls/sessions.sql

```diff
@@ -0,0 +1,69 @@
+-- Create a staging table.
+CREATE TEMPORARY TABLE `stage_sessions_{ds_no_dashes}`
+(
+  id                                 STRING    NOT NULL,
+  start_block_number                 INT64     NOT NULL,
+  start_block_timestamp              TIMESTAMP NOT NULL,
+  wallet_address                     STRING    NOT NULL,
+  contract_address                   STRING
+);
+
+
+-- Stage sessions for this execution date.
+INSERT INTO `stage_sessions_{ds_no_dashes}`
+SELECT
+  TO_HEX(MD5(CONCAT(
+    trace_id,
+    wallet_address,
+    contract_address,
+    block_timestamp
+  )))
+                    AS id,
+
+  block_number      AS start_block_number,
+  block_timestamp   AS start_block_timestamp,
+  wallet_address    AS wallet_address,
+  contract_address  AS contract_address
+FROM
+  `{destination_project_id}.{temp_dataset_name}.stage_root_call_traces_{ds_no_dashes}`
```

Comment:
Should be `{temp_dataset_name}.stage_root_call_traces_{ds_no_dashes}` instead of `{destination_project_id}.{temp_dataset_name}.stage_root_call_traces_{ds_no_dashes}`